File: app/src/main/java/com/bopr/android/smailer/MailMessage.java
package com.bopr.android.smailer;

import androidx.annotation.NonNull;

/**
 * Email message.
 *
 * @author Boris Pronin (<a href="mailto:boprsoft.dev@gmail.com">boprsoft.dev@gmail.com</a>)
 */
class MailMessage {

    final String id;
    final String subject;
    final String body;

    MailMessage(String id, String subject, String body) {
        this.id = id;
        this.subject = subject;
        this.body = body;
    }

    @Override
    @NonNull
    public String toString() {
        return "MailMessage{" +
                "id='" + id + '\'' +
                ", subject='" + subject + '\'' +
                ", body='" + body + '\'' +
                '}';
    }
}

File: app/src/main/java/com/bopr/android/smailer/MailMessage.kt
package com.bopr.android.smailer

import java.io.File

/**
 * Email message.
 *
 * @author Boris Pronin ([boprsoft.dev@gmail.com](mailto:boprsoft.dev@gmail.com))
 */
data class MailMessage(
        var id: String? = null,
        var subject: String? = null,
        var body: String? = null,
        var attachment: Collection<File>? = null,
        var recipients: String? = null,
        var replyTo: String? = null,
        var from: String? = null)